documentation/API-Readiness-Checklist.md

```diff
@@ -0,0 +1,16 @@
+# API Readiness Checklist
+| Nr | API release assets                           | alpha | release-candidate |  public-|release |
+|    |                                              |       |                   | initial | stable |
```

Comment:
a) I suggest to add a column where APIs can add their status or how it is intended to use the table?
b) Two lines in the header seem not to work in GitHub MarkDown, the following suggestion might work but I have not tested it.

```suggestion
| Nr | API release assets                           | alpha | release-candidate |  public-release<br>initial | public-release<br> | Status for <api-name> <version> in rX.Y |
```

Reply:
api-name, version and release can also go in a section before the table.